catroid/src/org/catrobat/catroid/bluetooth/ConnectBluetoothDeviceActivity.java

```diff
@@ -152,7 +152,7 @@ public void onReceive(Context context, Intent intent) {
 
 	private class ConnectDeviceTask extends AsyncTask<String, Void, BluetoothConnection.State> {
 
-		BluetoothConnection btConnection;
+		private BluetoothConnection btConnection;
```

Comment:
https://github.com/Catrobat/Catroid/pull/1216#discussion_r30360929
The check has to be excluded.